follow_fork/instructions.txt

```diff
@@ -0,0 +1,5 @@
+gcc -I /<path>/<to>/<undodb-xxxx>/undolr -L /<path>/<to>/<undodb-xxxx>/undolr -shared -fPIC follow_fork.c -o libfollowfork.so -l:libundolr_pic_x64.a -std=c99 -ldl
```

Comment:
Rename this to `README.md` and add a few more details on what this does and how to use it.

Reply:
Done